README.md

```diff
@@ -2,7 +2,7 @@
 
 > A simple Google News sitemap is generated on-the-fly for articles that were published in the last two days. Output is saved in cache or as a transient for fast reading and displaying on the front end.
 
-[![Support Level](https://img.shields.io/badge/support-beta-blueviolet.svg)](#support-level) [![Release Version](https://img.shields.io/github/release/10up/simple-google-news-sitemap.svg)](https://github.com/10up/simple-google-news-sitemap/releases/latest) ![WordPress tested up to version](https://img.shields.io/badge/WordPress-v6.0%20tested-success.svg) [![License](https://img.shields.io/github/license/10up/simple-google-news-sitemap.svg)](https://github.com/10up/simple-google-news-sitemap/blob/develop/LICENSE.md) [![Dependency Review](https://github.com/10up/simple-google-news-sitemap/actions/workflows/dependency-review.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/dependency-review.yml) [![Linting](https://github.com/10up/simple-google-news-sitemap/actions/workflows/lint.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/lint.yml) [![Test](https://github.com/10up/simple-google-news-sitemap/actions/workflows/test.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/test.yml)
+[![Support Level](https://img.shields.io/badge/support-beta-blueviolet.svg)](#support-level) [![Release Version](https://img.shields.io/github/release/10up/simple-google-news-sitemap.svg)](https://github.com/10up/simple-google-news-sitemap/releases/latest) ![WordPress tested up to version](https://img.shields.io/badge/WordPress-v6.3%20tested-success.svg) [![License](https://img.shields.io/github/license/10up/simple-google-news-sitemap.svg)](https://github.com/10up/simple-google-news-sitemap/blob/develop/LICENSE.md) [![Dependency Review](https://github.com/10up/simple-google-news-sitemap/actions/workflows/dependency-review.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/dependency-review.yml) [![Linting](https://github.com/10up/simple-google-news-sitemap/actions/workflows/lint.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/lint.yml) [![Test](https://github.com/10up/simple-google-news-sitemap/actions/workflows/test.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/test.yml)
```

Comment:
Maybe I'm missing something, but what changed here?

Reply:
WordPress tested up to version link: `https://img.shields.io/badge/WordPress-v6.3%20tested-success.svg`. Went from v6.0 to v6.3 in that URL

Reply:
Aha, right, that tiny bit that's here because this isn't on DOTORG.  Didn't show in the red/green compare.  Thanks for clarifying!